Dockerfile

```diff
@@ -76,22 +86,32 @@ ARG CONFIG="--prefix=/etc/nginx \
  --with-http_stub_status_module \
  --with-http_sub_module \
  --with-http_v2_module \
- --with-mail \
- --with-mail_ssl_module \
  --with-stream \
  --with-stream_realip_module \
  --with-stream_ssl_module \
  --with-stream_ssl_preread_module \
- --add-dynamic-module=/usr/src/njs/nginx \
- --add-dynamic-module=/usr/src/ngx_brotli"
+ --add-dynamic-module=/usr/src/ngx_brotli \
+ --add-dynamic-module=/usr/src/njs/nginx"
 
 RUN echo "Cloning nginx and building $NGINX_VERSION (rev $NGINX_COMMIT from '$NGINX_BRANCH' branch)" \
- && hg clone -b $NGINX_BRANCH --rev $NGINX_COMMIT https://hg.nginx.org/nginx-quic /usr/src/nginx-$NGINX_VERSION \
```

Comment:
Doesn't this break quic?

Reply:
Quic isn't turned on, ruled out as too slow IIRC